### ISSUE TITLE
Change Gauge logs format

### ISSUE THREAD

**Issue by nehashri** [getgauge/gauge#1391]
The current Gauge log format is 
```
11:52:03.140 Executing => gauge run /tests/python/specs/example.spec --simple-console
```
This log format doesn't have the date, process the output the message (gauge or plugin) and log level of the message.
This information will be useful while debugging an issue.
New Gauge log format to be implemented should be 
```
23-04-2019 11:52:03.140 [INFO] [Gauge] Executing => gauge run /tests/python/specs/example.spec --simple-console
```
i.e., `date time [log level] [gauge or plugin id] message`

**Comment by shubhamsc** [comment]
New Gauge log format will be show as -
```
23-04-2019 11:52:03.140 [Gauge] [INFO] Executing => gauge run /tests/python/specs/example.spec --simple-console
```

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the version: ```1.0.6.nightly-2019-05-17 ```
